components/02-elements/breadcrumbs/breadcrumbs.hbs

```diff
@@ -1,21 +1,26 @@
-<div class="breadcrumbs {{ class }}" {{{ attributes }}}>
+<ol
```

Comment:
one more thing: can we add as a container `nav` element? with `<nav><ol></ol></nav>` we use navigation landmark to make navigation functionality. is something like simple "menu" and move please the attributes: to this nav container (instead of the list)

Reply:
Done